### ISSUE TITLE
Accessibility Audit - Links does not have a 'discernible name'

### ISSUE THREAD

**Issue by Oliversw** [onwordi/onwordi.github.io#1]
According to Chrome's accessibility audit: "Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users."

Failing element:
```
<a href="https://github.com/onwordi/onwordi.github.io">
```

Putting a description as alt text for the image should fix the issue.
